src/admin.c

```diff
@@ -253,6 +267,85 @@ main(int argc, char** argv)
       errx(1, "Using the root account is not allowed");
    }
 
+   // and she cannot use "local" and "remote" connections at the same time
+   if (configuration_path != NULL && remote_connection)
+   {
+      errx(1, "Use either -c or -h/-p to define endpoint");
+   }
+
+   if (argc <= 1)
+   {
+      usage();
+      exit(1);
+   }
+
+   size = sizeof(struct main_configuration);
+   if (pgagroal_create_shared_memory(size, HUGEPAGE_OFF, &shmem))
+   {
+      errx(1, "Error creating shared memory");
+   }
+   pgagroal_init_configuration(shmem);
+
+   if (configuration_path != NULL)
+   {
+      ret = pgagroal_read_configuration(shmem, configuration_path, false);
+      if (ret == PGAGROAL_CONFIGURATION_STATUS_FILE_NOT_FOUND)
+      {
+         errx(1, "Configuration not found: <%s>", configuration_path);
+      }
+      else if (ret == PGAGROAL_CONFIGURATION_STATUS_FILE_TOO_BIG)
+      {
+         errx(1, "Too many sections in the configuration file <%s>", configuration_path);
+      }
+
+      if (logfile)
+      {
+         config = (struct main_configuration*)shmem;
+
+         config->common.log_type = PGAGROAL_LOGGING_TYPE_FILE;
+         memset(&config->common.log_path[0], 0, MISC_LENGTH);
+         memcpy(&config->common.log_path[0], logfile, MIN(MISC_LENGTH - 1, strlen(logfile)));
+      }
+
+      if (pgagroal_start_logging())
+      {
+         errx(1, "Cannot start the logging subsystem");
+      }
+
+      config = (struct main_configuration*)shmem;
+   }
+   else
+   {
+      ret = pgagroal_read_configuration(shmem, PGAGROAL_DEFAULT_CONF_FILE, false);
+      if (ret != PGAGROAL_CONFIGURATION_STATUS_OK)
+      {
+         if (!remote_connection)
+         {
+            errx(1, "Host (-h) and port (-p) must be specified to connect to the remote host");
```

Comment:
Is `pgagroal-admin` supposed to be able to connect to the remote host? It doesn't even contain the arguments the message is claiming to use. I suspect there is extra copy and paste here.

Reply:
pgagroal-admin is local only, pgagroal-cli can do remote